client/src/store/new-request-module.ts

```diff
@@ -2313,26 +2314,27 @@ export class NewRequestModule extends VuexModule {
       }
 
       const requestData = data && await this.addRequestActionComment(data)
+      const response = requestData && await axios.put(`/namerequests/${nrId}`, requestData, {
+        headers: { 'Content-Type': 'application/json' }
+      })
 
-      try {
-        const response = requestData && await axios.put(`/namerequests/${nrId}`, requestData, {
-          headers: {
-            'Content-Type': 'application/json'
-          }
-        })
+      if (response?.data) {
         this.setNrResponse(response.data)
```

Comment:
This was causing "putNameReservation() = Error: undefined is not an object (evaluating 'response.data')."